python/railjson_generator/railjson_generator/schema/infra/operational_point.py

```diff
@@ -9,13 +9,15 @@ class OperationalPoint:
     label: str
     trigram: str
     parts: List
+    weight: int
     uic: int
 
     def __init__(self, label: str, trigram: Optional[str] = None, uic: int = 0):
         self.label = label
         self.trigram = trigram or label[:3].upper()
         self.parts = list()
         self.uic = uic
+        self.weight = None
```

Comment:
nitpick: Would it make sense to do that instead?

```suggestion
    def __init__(self, label: str, trigram: Optional[str] = None, uic: int = 0, weight: Optional[int] = None):
        self.label = label
        self.trigram = trigram or label[:3].upper()
        self.parts = list()
        self.uic = uic
        self.weight = weight
```